pkg/controller/upload-controller.go

```diff
@@ -238,6 +241,13 @@ func (r *UploadReconciler) reconcilePVC(log logr.Logger, pvc *corev1.PersistentV
 		if podRestarts > pvcAnnPodRestarts {
 			anno[AnnPodRestarts] = strconv.Itoa(podRestarts)
 		}
+
+		if pod.Status.ContainerStatuses[0].State.Terminated != nil &&
+			pod.Status.ContainerStatuses[0].State.Terminated.ExitCode == 0 {
+			if strings.Contains(pod.Status.ContainerStatuses[0].State.Terminated.Message, "Preallocation applied") {
```

Comment:
I am not sure we have a const for common messages, but this is a good candidate, it is repeated in a few places

Reply:
Good point.